doc/installation/system/inifile.rst

```diff
@@ -226,13 +226,17 @@ privacyIDEA Nodes
 -----------------
 
 privacyIDEA can run in a redundant setup. For statistics and monitoring purposes you
-can give these different nodes, dedicated names.
+can give these different nodes dedicated names.
 
-``PI_NODE`` is a string with the name of this very node. ``PI_NODES`` is a list of
-all available nodes in the cluster.
+``PI_NODE`` is a string with the name of this very node. At the startup of
+the application, an installation specific unique ID will be used to tie the
+node name to an installation. A distinct ID can be given to an application with
```

Comment:
"A distinct ID can be given".

passive sentences are unclear. Who and how?
It think this means: instead of autogenerating the ID, the administrator (you) can manually define an ID by using PI_UUID.

Reply:
Done in c0c5f2d5e